#Request Fields
std_msgs/String command
std_msgs/String topic
std_msgs/String fileName
---
#Response Fields
std_msgs/Empty empty